### DISCUSSION TITLE
Is it possible to make a node template with a set of attributes?

### DISCUSSION THREAD

**Discussion by nerkuda** [freeplane/freeplane#959] [Help] [answered]
Hi, everyone

How do I create a node template with the necessary attributes?

For example, when I select an arbitrary node and apply a task template, attributes from the template are automatically added to this node?

**Comment by euu2021** [comment, marked as answer]
This is one way:
1-Create a style to be the template. Insert the attributes in the style:
![image](https://user-images.githubusercontent.com/77707706/213883050-830fcaa7-6da0-48ee-bfd3-35de61720608.png)

2-Apply the style to the node:

3- Use Edit->Node properties->Attributes from style

Result:
![image](https://user-images.githubusercontent.com/77707706/213883107-3e4aec59-fccd-4813-bf68-3f6570123289.png)

**Reply by nerkuda** [reply]
Thanks @euu2021 .
It wasn't obvious to me. Thanks to you, I can now cope with my task
